docs-gen/content/rule_set/overlay.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Overlay"
+date: 2019-08-04T12:46:30+02:00
+weight: 7
+---
+
+VSS defines the standard catalog for vehicle signals independent of the used protocol and environment.
+In order to adopt the specification we realize that certain additions and modifications to the standard catalog are necessary.
+**VSS Overlay** are meant to bring you a standardized way of handling those changes.
+
+The following features with the intended usage patterns are currently supported:
+1. **Adding new nodes:** By adding nodes the standard catalog can be extended with proprietary concepts.
+1. **Changing the value of existing metadata:** The standard catalog defines metadata based on what is assumed to be an average vehicle. 
+Configurations may differ slightly (e.g. the instantiation of number of available seats), or special situations that require a (limited) modification of existing metadata.
+1. **Adding new key/value pairs as additional metadata:** 
```

Comment:
That is true, but only after https://github.com/COVESA/vss-tools/pull/173 is merged